templates/scenarios/js/m365-tab/teamsapp.yml.tpl

```diff
@@ -96,22 +110,36 @@ deploy:
 
 # Triggered when 'teamsfx publish' is executed
 publish:
-  - uses: teamsApp/validateManifest # Validate using manifest schema
+  # Validate using manifest schema
+  - uses: teamsApp/validateManifest
```

Comment:
Not related to lint format, but it seems not all templates have this `validateManifest`?

Reply:
Thanks, let me check.